docs/data/material/getting-started/templates/dashboard/components/MenuButton.tsx

```diff
@@ -19,7 +21,7 @@ export default function MenuButton({
         '& .MuiBadge-badge': { right: 2, top: 2 },
       }}
     >
-      <IconButton size="small" {...props} />
+      <IconButton size="small" aria-label={ariaLabel} {...props} />
```

Comment:
You should be able to use `aria-label` without passing it as a prop, given we're spreading the props with `{...props}` — have you tried this?